langchain4j/src/test/java/dev/langchain4j/service/AiServicesSystemAndUserMessageConfigsTest.java

```diff
@@ -125,10 +125,10 @@ void test_system_message_configuration_1() {
         // when-then
         assertThat(aiService.chat1("Country: Germany"))
                 .containsIgnoringCase("Berlin");
-        verify(chatLanguageModel).generate(asList(
+        verify(chatLanguageModel).chat(ChatRequest.builder().messages(
                 systemMessage("Given a name of a country, answer with a name of it's capital"),
```

Comment:
`its` not `it's capital`

Reply:
Damn, this mistake propagated to a lot of classes, I do not want to fix it here becasue PR will grow even bigger